VideoLocker/src/main/java/org/edx/mobile/view/VideoListActivity.java

```diff
@@ -251,23 +250,14 @@ public void updateProgress() {
     }
 
     public void showCheckBox() {
-        if (myVideosFlag) {
-            AppConstants.myVideosDeleteMode = true;
-        } else {
-            AppConstants.videoListDeleteMode = true;
-        }
+        AppConstants.myVideosDeleteMode = true;
```

Comment:
Note that using global static variables to share state between `Activity` and `Fragment` as is implemented here is an extremely hacky and unadvisable approach, and should be changed at some point. There are also other issues with the logic for retaining the `Fragment` here, not to mention that every method is wrapped in a generic try catch block that I didn't remove from the unmodified methods for fear that they might mask some hard-to-resolve crash.